### DISCUSSION TITLE
Plotting configuration in paralell

### DISCUSSION THREAD

**Discussion by gonbressan** [Chia-Network/chia-blockchain#5866] [Support]
Hi, I think that is not clear if the plotting configuration is per plot or for all when a paralell plotting is configured.

Can you specify this to me whit 100% of security?

![Captura de pantalla de 2021-05-23 08-41-29](https://user-images.githubusercontent.com/20097548/119259036-b4f54100-bba2-11eb-9840-8551bf711546.png)

Thanks.

**Comment by padik11** [comment]
For every plot

**Comment by BrandtH22** [comment]
@gonbressan , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
